helloworld/Jenkinsfile

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/groovy
+node('redhatdistortion-maven') {
+    stage 'Checkout'
+    checkout scm
+
+    stage 'Build'
+    sh 'cd helloworld && mvn clean compile'
+
+    stage 'Run Unit Tests'
+    sh 'cd helloworld && mvn test'
+
+    stage 'Package'
+    sh 'cd helloworld && mvn package'
+
+    stage 'Archive artifact'
+    archive 'helloworld/target/*.war'
+
+    stage 'Create Image'
+    sh 'oc login https://kubernetes.default/ -u openshift-dev -p devel --insecure-skip-tls-verify=true'
```

Comment:
True. I made it initially just to work with ADB and beacause I am busy a lot with other important issues.. I have been looking into service accounts after your comment and I think I can build in something like `oc --token=$(cat /run/secrets/kubernetes.io/serviceaccount/token) --certificate-authority=/run/secrets/kubernetes.io/serviceaccount/ca.crt ....` hopefully tomorrow.